experiments/yarn/src/main/scala/org/apache/gearpump/experiments/yarn/master/ServicesLauncherActor.scala

```diff
@@ -0,0 +1,24 @@
+package org.apache.gearpump.experiments.yarn.master
```

Comment:
I think it would be better to keep master separate from YarnAM for simplicity especially if master is HA. Putting the ui in the YarnAM makes sense since if the YarnAM dies we know the trackingURL will also die.